client/notifications/style.scss

```diff
@@ -216,4 +216,18 @@ div.reader-notifications__panel {
 		}
 	}
 }
-
+@media only screen and (max-width: 783px) {
```

Comment:
do we have existing sass mixins/etc for breakpoints?

Reply:
We do, but none of them seem to be accessible to this scss file. 783px seems to be hard coded in a number of different spots. Not sure I feel comfortable refactoring stuff in this PR.

Reply:
cool 👍